src/status_im/data_store/realm/core.cljs

```diff
@@ -70,12 +70,7 @@
   ;; delete the realm file if its schema version is higher
   ;; than existing schema version (this means the previous
   ;; install has incompatible database schemas)
-  (if-let [current-version (realm-version file-name encryption-key)]
-    (if (> current-version
-           (apply max (map :schemaVersion base/schemas)))
-      (reset-realm file-name schemas encryption-key)
-      (migrate-realm file-name schemas encryption-key))
-    (reset-realm file-name schemas encryption-key)))
+  (migrate-realm file-name schemas encryption-key))
```

Comment:
could this be done in a separate commit that we merge right after 0.19 cut ?

Reply:
Yes, no probs, it will try to reset the db, but as long as we are aware of it I see no problems